api/src/play_asset_pack_manager.gd

```diff
@@ -301,7 +305,11 @@ func cancel_asset_pack_request(pack_name : String) -> bool:
 	var updated_asset_pack_state : PlayAssetPackState = updated_asset_pack_states.get_pack_states()[pack_name]
 	var updated_asset_pack_status = updated_asset_pack_state.get_status()
 	
-	return updated_asset_pack_status == AssetPackStatus.CANCELED
+	var cancellation_success = updated_asset_pack_status == AssetPackStatus.CANCELED
```

Comment:
Is this necessary? When you cancel a request, do you get a new state update with the new canceled status?

Reply:
This line is needed since as I had tested on real machines, assetPackStateUpdated signal won't be trigger if we call cancel() on an asset pack. I also added some comments describing this situation.